multiversx_sdk_core/custom_tokens.py

```diff
@@ -0,0 +1,49 @@
+from multiversx_sdk_core.codec import decode_unsigned_number
+from multiversx_sdk_core.errors import InvalidTokenIdentifierError
+
```

Comment:
Now I see. Perhaps we should rename the file to `tokens`, in `sdk-specs`, as well?

Reply:
Renamed to `tokens.py`. Will follow up with a PR on `sdk-specs`.